### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by borusova-ljubov**

### CODE REVIEW

File: calc.sub 321.side [hunk 1]
```diff
@@ -685,7 +612,7 @@
     "persistSession": false,
     "parallel": false,
     "timeout": 300,
-    "tests": ["f0fe93f0-84d7-42c7-bfe3-a0f2f8a88ad3"]
+    "tests": []
```
Comment (getman) [inline calc.sub 321.side:615]: Comment 1

File: calc.s.side [hunk 2]
```diff
@@ -679,79 +679,6 @@
       "targets": [],
       "value": "-864197531986420"
     }]
-  }, {
-    "id": "d768be07-db5a-4fcc-a87c-6ba1c0c580af",
-    "name": "sub.0-0",
-    "commands": [{
```
Comment (getman) [inline calc.s.side:685]: Comment 2

**Review by getman — CHANGES_REQUESTED** [review]

File: calc.sub 321.side (hunk 1 above)
Comment (borusova-ljubov) [reply calc.sub 321.side:615]: Commint -1

File: calc.s.side (hunk 2 above)
Comment (borusova-ljubov) [reply calc.s.side:685]: Commint3

**Review by getman — APPROVED** [review]

**Pull request merged**